portality/templates/email/editor_application_assigned_group.txt

```diff
@@ -1,6 +1,6 @@
 Dear {{ editor }},
 
-A new application or a reapplication for the journal "{{ journal_name }}" has been
+A new application or a update request for the journal "{{ journal_name }}" has been
```

Comment:
**an** update request?

Reply:
fixed.  I also checked the rest of the code for any similar grammatical errors.